sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPlugin.scala

```diff
@@ -109,11 +112,18 @@ object ScalaJSPlugin extends AutoPlugin {
 
     // All our public-facing keys
 
-    val fastOptJS = TaskKey[Attributed[File]]("fastOptJS",
+    val fastOptJS = TaskKey[File]("fastOptJS",
```

Comment:
Uh oh. This will probably break half of the builds in the world. And it is not warranted. I strongly suggest we keep an `Attributed[File]`, even if it has no attribute.